src/pynwb/file.py

```diff
@@ -87,8 +95,13 @@ def __init__(self, **kwargs):
                        "date_of_birth",
                        "strain")
         args_to_set = popargs_to_dict(keys_to_set, kwargs)
-        kwargs['name'] = 'subject'
-        super().__init__(**kwargs)
+        super().__init__(name="subject", **kwargs)
+
+        if args_to_set["age__reference"] is not None:
+            if args_to_set["age__reference"] not in ("birth", "gestational"):
+                raise ValueError("age__reference must be 'birth' or 'gestational'.")
+            if args_to_set["age"] is None:
+                raise ValueError("If age__reference is provided, age must also be provided.")
```

Comment:
Since ``age__reference`` has a default value, would this not always raise an error if ``age`` is not provided by the user?

Reply:
oh yeah good point